kaitai_utils.py

```diff
@@ -45,6 +47,28 @@ def _get_metadata():
         return yaml.safe_load(file)['meta']
 
 
+def search_process_key(object):
+    if type(object) is dict:
+        for key in object.keys():
+            if 'process' in object:
+                return True
+            if type(object[key]) is list or type(object[key]) is dict:
```

Comment:
My Python is not great, but is it possible to do this?
```suggestion
            if type(object[key]) in [list, dict]:
```